src/components/Input/Input.d.ts

```diff
@@ -0,0 +1,86 @@
+import {IconType} from '@Components/Icon';
+
+export interface InputProps {
+
+    /**
+		 * Name of the icon from the icon map
+		 */
```

Comment:
Can we use comments as `// test-comment` ??

Reply:
we can but using multiline comments help us in creating documentation in storybook
![image](https://user-images.githubusercontent.com/102018792/196136827-3c5beab7-fa60-4df8-9d99-cbcb962426bf.png)